### PR TITLE
chore: move auth and settings back into the template

### PR DESCRIPTION
**Pull request by redreceipt**
motivation for this PR was so we can use local images in the auth components instead of icons. turned out to be a lot of work.

I know @calebpanza I'm such a hypocrite 😔 the bulk of these changed files are moving `ui-authentication` and `UserSettings` back to the template and then getting tests working correctly. There's some external pkg that needs to be mocked in `ui-kit` or `ui-connected` that was breaking tests so instead of going through every import to find, I simply mocked those pkgs and the pieces we're using. Which turns out to actually produce much cleaner, faster tests.

### CODE REVIEW

File: templates/mobile/src/auth/Identity/VerificationConnected.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import renderer from 'react-test-renderer';
+import { LoginContext } from '../LoginProvider';
+import { AuthContext } from '../Provider';
+import VerificationConnected from './VerificationConnected';
+
+//jest.useFakeTimers();
+
```
Comment (redreceipt) [inline templates/mobile/src/auth/Identity/VerificationConnected.test.js:7]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -31,11 +31,18 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
           node-version: 14
+      - uses: ruby/setup-ruby@v1
+        with:
+          working-directory: templates/mobile
```
Comment (redreceipt) [inline .github/workflows/ci.yml:39]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -31,11 +31,15 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
           node-version: 14
       - run: yarn
       - run: ./scripts/test-all.sh
+      - run: |
+          yarn --ignore-scripts
```
Comment (vinnyjth) [inline .github/workflows/ci.yml:40]: why `yarn` twice?

Comment (vinnyjth) [reply .github/workflows/ci.yml:40]: oh word, thanks!

**Pull request merged**